app/src/polkadot_types.c

```diff
@@ -65,19 +73,55 @@ parser_error_t _readCompactSubId(parser_context_t *c, pd_CompactSubId_t *v) {
     return _readCompactInt(c, v);
 }
 
-parser_error_t _readAccountId(parser_context_t *c, pd_AccountId_t *v) {
+parser_error_t _readAccountId(parser_context_t *c, pd_AccountId_t *v) {   
```

Comment:
Let's remove all these ending spaces so the diffs are smaller

Reply:
Ok, those should be fixed now